core/backend/src/IModelJsFs.ts

```diff
@@ -59,8 +59,8 @@ export class IModelJsFs {
   /** Make a copy of a file */
   public static copySync(src: string, dest: string, opts?: any): void { fs.copySync(src, dest, opts); }
 
-  /** Get the file and directory names in the specified directory. Excludes "." and "..". */
-  public static readdirSync(pathname: string): string[] { return fs.readdirSync(pathname); }
+  /** Gets the file and directory names in the specified directory. Excludes "." and "..". Returns an empty array if the specified directory does not exist. */
+  public static readdirSync(pathname: string): string[] { return fs.existsSync(pathname) ? fs.readdirSync(pathname) : []; }
```

Comment:
I will resolve this comment after I have tested on mobile. Testing on mobile will take some time.

Reply:
I have confirmed that this fix works on mobile in display-test-app.